ui/src/scss/_sizing.scss

```diff
@@ -186,6 +186,9 @@ $abbrev in (width: w, height: h) {
 .min-w-250p {
   min-width: 250px;
 }
+.h-100-60{
```

Comment:
use rem value, limit the usage of px units
h-100-60 class name can be rephrase to convey better

Reply:
done used rem and rephrased h-100-n60